src/app/components/SubjectHeading/SubjectHeadingShow.jsx

```diff
@@ -111,12 +112,21 @@ class SubjectHeadingShow extends React.Component {
     return onMainPath;
   }
 
+  findBibCount(headings, uuid) {
```

Comment:
Let's include the entire main heading data in the API response, instead of just the label.

Reply:
Not sure I understand your comment. Are you saying this should be changed in the back end?

Reply:
Changed this on the backend.